src/main/java/org/spongepowered/api/entity/living/monster/Wither.java

```diff
@@ -34,7 +35,7 @@
 /**
  * Represents the Wither.
  */
-public interface Wither extends Monster, ProjectileSource, Boss, Aerial {
+public interface Wither extends Monster, ProjectileSource, Boss, Aerial, FusedExplosive {
```

Comment:
Yeah, a Wither explodes after it's been "invulnerable" for some time after it spawns.